.gitignore

```diff
@@ -30,6 +30,7 @@ hs_err_pid*
 node_modules/
 npm-debug.log
 yarn-error.log
+package-lock.json
```

Comment:
According to [npm docs](https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md) package-lock.json is meant to be committed

Reply:
@ziccardi this is the react native client and package-lock actually breaks it.